easy-coding-standard.neon

```diff
@@ -19,3 +19,5 @@ parameters:
     skip:
         PHP_CodeSniffer\Standards\Generic\Sniffs\NamingConventions\CamelCapsFunctionNameSniff:
             - */tests/**
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\Classes\ValidClassNameSniff:
+            - */src/**
```

Comment:
My original thought was that because there were so many (13 of the >20 classes) that when I had time to dig into it, I *might* discover that the classes involved were all internal classes, and thus I could refactor their names without a BC break.  That's why I went with a blanket `src` here.